src/components/TodoList.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+// import classNames from 'classnames';
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments). Fix all cases if present